data/registry/instrumentation-python-watsonx.yml

```diff
@@ -7,11 +7,12 @@ tags:
   - instrumentation
   - IBM
   - Watsonx
+  - Instana
 license: Apache 2.0
 description:
   Instrumentation for IBM Watsonx LLMs (prompts logging, tokens, etc.)
 authors:
-  - name: Guangya Liu
+  - name: IBM Instana
     email: gyliu@ibm.com
```

Comment:
do you want to update the email as well?

Reply:
Done